contents/layouts/headers/with-tabs/index.tsx

```diff
@@ -80,32 +80,67 @@ const WithTabs: FC = () => {
                 w="6"
                 rounded="full"
               />
-              <Text color={["black", "white"]}>taroj1205</Text>
+              <Text
+                color={["black", "white"]}
+                fontSize="md"
+                fontWeight="medium"
+              >
+                taroj1205
+              </Text>
             </MenuButton>
 
             <MenuList>
-              <MenuItem icon={<HeartIcon color="pink.500" />}>
+              <MenuItem fontSize="sm" icon={<HeartIcon color="pink.500" />}>
                 Liked posts
               </MenuItem>
-              <MenuItem icon={<StarIcon color="yellow.500" />}>
+              <MenuItem fontSize="sm" icon={<StarIcon color="yellow.500" />}>
                 Saved posts
               </MenuItem>
-              <MenuItem icon={<MessageSquareIcon color="blue.500" />}>
+              <MenuItem
+                fontSize="sm"
+                icon={<MessageSquareIcon color="blue.500" />}
+              >
                 Your comments
               </MenuItem>
-              <MenuGroup label="Settings">
-                <MenuItem icon={<SettingsIcon />}>Account settings</MenuItem>
-                <MenuItem icon={<ArrowLeftRightIcon />}>
+              <MenuGroup
+                label="Settings"
+                __css={{
+                  ".ui-menu__item--group-label": {
+                    fontSize: "xs",
+                    fontWeight: "medium",
+                  },
+                }}
```

Comment:
I couldn't specify a style for the group-label of menu-item, so I'm specifying it directly,
 I'll fix it later.

Reply:
https://github.com/yamada-ui/yamada-ui/blob/e7c8bd5020102c240f6b1dbc2855e34049c5f62f/packages/components/menu/src/menu-group.tsx#L30

Reply:
A custom theme solved the problem.